kviz/dense.py

```diff
@@ -51,7 +81,29 @@ class DenseGraph():
 
     """
 
-    def __init__(self, model):
+    def __init__(self, model, input_color='#3498db', inner_color='#2ecc71', output_color='#3498db',
```

Comment:
Please provide a descriptive doctring. Each input parameter ought to have some basic description, even if the names are relatively descriptive themselves.

Reply:
This is helpful (1) for refactoring down the line, because a contributor can read the code and understand which visual aspect each variable pertains to and (2) for users to read the code as its own documentation.

Reply:
Doc string for __init__ is added. Also, I include a reference for some usable shapes: https://graphviz.org/doc/info/shapes.html
Some changes are also added:
(1) now the filled color of the output layer can be different from that of the input layer.
(2) when creating a colormap, potentially there is more freedom choosing the size of the colormap as well as the range of color. See create_colormap() (line 29 in dense.py) for details; it now accepts 2 more inputs N and step and they are with default values.